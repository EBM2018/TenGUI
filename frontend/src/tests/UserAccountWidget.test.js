import ActTextCheck from '../react/Widgets/UserAccountWidget/UserAccountWidget';
import React from 'react';
import renderer from 'react-test-renderer';

test ('derp',()=>{
    expect('True').toBeTruthy;
});
