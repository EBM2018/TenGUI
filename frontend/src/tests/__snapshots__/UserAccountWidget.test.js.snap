// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does this render 1`] = `
<figure
  className="image is-64x64"
>
  <img
    alt="MonProfil"
    src="../../../images/PhotoProfil.png"
  />
  <p>
    
  </p>
</figure>
`;
